include/my_rdtsc.h

```diff
@@ -152,7 +152,7 @@ C_MODE_START
 */
 static inline ulonglong my_timer_cycles(void)
 {
-# if __has_builtin(__builtin_readcyclecounter) && !defined (__aarch64__)
+# if __has_builtin(__builtin_readcyclecounter) && !defined (__aarch64__) && !defined(__riscv)
```

Comment:
This should be gated on __linux__, other operating systems such as FreeBSD don't make this inaccessible. Quite surprising to see that x86 rdtsc is permitted but the aarch64 counters are not.

Reply:
Noted, adjusted. Seems the Aarch64 was removed for performance reasons instead rather than Kernel limitation. Oops mea cupla.